certified-connectors/Nosco/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1543 @@
+{
+  "schemes" : [ "https" ],
```

Comment:
Please update apiDefinition.swagger.json file as in this path. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json

Reply:
Please advise on how the file should be updated according to the template. The swagger definition passes all validation checks, meaning all required fields are present and correct.

Reply:
I've updated the file to ensure the top-level keys match the order found in the template.